bpf-gpl/conntrack.h

```diff
@@ -127,7 +127,7 @@ struct ct_ctx {
 	__u16 dport;
 	__u16 orig_dport;
 	struct tcphdr *tcp;
-	__be32 nat_tun_src;
+	__be32 tun_ip;
```

Comment:
Please add comment explaining which types of NAT entry have this set and what it is set to in each case.